src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h

```diff
@@ -858,48 +898,60 @@ namespace OpenMS
     "A:C15H18N5O6P1;NM+A-NH3-H2O"
   };
 
-  static constexpr std::array<const char*, 34> fragments_RNA_FA
+  static constexpr std::array<const char*, 46> fragments_RNA_FA
   {
-      "U:C3O;C3O",
-      "U:C4H4N2O2;U'",
-      "U:C4H2N2O1;U'-H2O",
-      "U:C9H13N2O9P1;U",
-      "U:C9H11N2O8P1;U-H2O",
-      "U:C9H12N2O6;U-HPO3",
-      "U:C9H10N2O5;U-H3PO4",
+    "U:C3O;C3O",
+    "U:C4H4N2O2;U'",
+    "U:C4H2N2O1;U'-H2O",
+    "U:C9H13N2O9P1;U",
+    "U:C9H11N2O8P1;U-H2O",
+    "U:C9H12N2O6;U-HPO3",
+    "U:C9H10N2O5;U-H3PO4",
 
     "G:C;FA",
     "G:C5H5N5O;G'",
-    "G:C6H5N5O;G'+FA",
-    "G:C7H5N5O;G'+2FA",
     "G:C10H14N5O8P;G",
+    "G:C6H5N5O;G'+FA",
     "G:C11H14N5O8P;G+FA",
+    "G:C11H12N5O7P;G+FA-H2O",
+    "G:C11H11N5O4;G+FA-H3PO4",
     "G:C11H13N5O5;G+FA-HPO3",
+    "G:C7H5N5O;G'+2FA",
     "G:C12H14N5O8P;G+2FA",
     "G:C12H13N5O5;G+2FA-HPO3",
+    "G:C12H11N5O4;G+2FA-H3PO4",
+    "G:C12H13N5O5;G+2FA-HPO3",
```

Comment:
same as in line 921?

Reply:
Ohh.. should be one of this "G:C12H13N5O6P;G+2FA-H2O",

Reply:
k maybe double check it once more. I know it is super easy to make small mistakes here :)